### ISSUE TITLE
Failed to build Golum from master branch

### ISSUE THREAD

**Issue by mikechris** [trivago/gollum#76]
PR submitted: [https://github.com/trivago/gollum/pull/75](https://github.com/trivago/gollum/pull/75)
```
GoWork/src/github.com/trivago/gollum$
go build
# github.com/trivago/gollum/consumer
consumer/http.go:170: undefined: body
consumer/http.go:170: undefined: err
consumer/http.go:171: undefined: err
consumer/http.go:173: undefined: err in err.Error
consumer/http.go:178: undefined: body

```

**Comment by andygrunwald** [comment]
But why is travis successful ? 

https://travis-ci.org/trivago/gollum/jobs/215181312

**Comment by mikechris** [comment]
In my Fixed PR yes build ok, but here is failing: https://travis-ci.org/trivago/gollum/builds/211696250

**Comment by arnecls** [comment]
As mentioned in the pull request - this bug is already fixed in 0.4.5dev.
I'll leave this open until the release next week.

**Comment by arnecls** [comment]
0.4.5 has been merged
